GmsCompat crash

```
type: crash
osVersion: google/shiba/shiba:14/UQ1A.240105.004/2024012600:user/release-keys
package: com.google.android.gms:240213038
process: com.google.android.gms.persistent
processUptime: 77255142 + 237 ms
installer: com.android.vending
GmsCompatConfig version: 93

java.lang.SecurityException: registerVibratorStateListener: Neither user 10141 nor current process has android.permission.ACCESS_VIBRATOR_STATE.
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3066)
	at android.os.Parcel.createException(Parcel.java:3050)
	at android.os.Parcel.readException(Parcel.java:3026)
	at android.os.Parcel.readException(Parcel.java:2968)
	at android.os.IVibratorManagerService$Stub$Proxy.registerVibratorStateListener(IVibratorManagerService.java:323)
	at android.os.SystemVibratorManager$SingleVibrator.addVibratorStateListener(SystemVibratorManager.java:286)
	at android.os.SystemVibratorManager$SingleVibrator.addVibratorStateListener(SystemVibratorManager.java:262)
	at dvwp.b(:com.google.android.gms@240213038@24.02.13 (190400-599553831):50)
	at dwbl.a(:com.google.android.gms@240213038@24.02.13 (190400-599553831):3)
	at dwby.r(:com.google.android.gms@240213038@24.02.13 (190400-599553831):20)
	at dwbo.i(:com.google.android.gms@240213038@24.02.13 (190400-599553831):3)
	at dwbl.d(:com.google.android.gms@240213038@24.02.13 (190400-599553831):26)
	at dwby.d(:com.google.android.gms@240213038@24.02.13 (190400-599553831):8)
	at dwbr.run(:com.google.android.gms@240213038@24.02.13 (190400-599553831):10)
	at akki.c(:com.google.android.gms@240213038@24.02.13 (190400-599553831):50)
	at akki.run(:com.google.android.gms@240213038@24.02.13 (190400-599553831):97)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at akpt.run(:com.google.android.gms@240213038@24.02.13 (190400-599553831):8)
	at java.lang.Thread.run(Thread.java:1012)
	Suppressed: ebdl: 
		at tk_trace.location-_onReceive_scheduled_t(Unknown Source:0)
Caused by: android.os.RemoteException: Remote stack trace:
	at android.app.ContextImpl.enforce(ContextImpl.java:2395)
	at android.app.ContextImpl.enforceCallingOrSelfPermission(ContextImpl.java:2423)
	at com.android.server.vibrator.VibratorManagerService.registerVibratorStateListener(VibratorManagerService.java:312)
	at android.os.IVibratorManagerService$Stub.onTransact(IVibratorManagerService.java:171)
	at android.os.Binder.execTransactInternal(Binder.java:1387)
```

--- a/gmscompat_config
+++ b/gmscompat_config
@@ -203,7 +203,7 @@ getVoiceMailNumber nullString
 [[versionMap]]
 # section name is gmscompat version, section contents are max supported versions of GmsCore and Play Store
 [1000]
-240599999 83959999
+240599999 83969999
 
 [[stubs_12.1]]
 

--- a/gmscompat_config
+++ b/gmscompat_config
@@ -155,6 +155,9 @@ disable false
 [android.os.DropBoxManager]
 getNextEntry null
 
+[android.os.SystemVibratorManager$SingleVibrator]
+addVibratorStateListener void
+
 [android.os.UserManager]
 getSeedAccountOptions null
 getUserName String Owner

